apps/prairielearn/src/schemas/schemas/questionOptionsMultipleChoice.json

```diff
@@ -19,7 +19,8 @@
       "type": "array",
       "items": {
         "type": "string"
-      }
+      },
+      "minItems": 1
```

Comment:
Is this correct? I'm less sure about this being wrong than I am for `apps/prairielearn/src/schemas/schemas/questionOptionsMultipleTrueFalse.json`.

Reply:
This is correct:

```
var trueIndex = _(perm).indexOf(0);
    var trueAnswer = {
      key: answers[trueIndex].key,
      text: answers[trueIndex].text,
    };
```

from `apps/prairielearn/public/localscripts/calculationQuestion/MCQServer.js` requires this to be at least 1.